parsedatetime/__init__.py

```diff
@@ -863,7 +898,166 @@ def _quantityToReal(self, quantity):
         except KeyError:
             pass
 
-        return 0.0
+        return 0.0'''
+
+    def number_formation(self, number_words):
```

Comment:
What is the purpose of this function? Can you please write some explanation?

Reply:
reason for this function is to actuall create the three digit numbers from the number words. I inserted a comment